src/core/RecyclerListView.tsx

```diff
@@ -110,6 +110,7 @@ export interface RecyclerListViewProps {
     scrollViewProps?: object;
     applyWindowCorrection?: (offsetX: number, offsetY: number, windowCorrection: WindowCorrection) => void;
     onItemLayout?: (index: number) => void;
+    onSizeChange?: (rlvDimension: Dimension, contentDimension: Dimension) => void;
```

Comment:
You can use contentSizeChanged event of scrollview instead

Reply:
@naqvitalha does RLV's `_onSizeChanged` get called on content size change or just container size change?

Reply:
@ananyachandra14 `contentSizeChanged ` this gets called only for ScrollView container size has changed.
@naqvitalha We had to add this for any size change (RVL + ScrollView container).. and to know if the RLV is scrollable or not.

Reply:
@manishPatwari can we add a callback for both RLV size change and content size change and force rerender in both?

Reply:
This change is for the same. RLV change hight is computed in onSize itself.

Reply:
But I'm not sure if this will cover the scenario where our RLV size remains the same but we dynamically increase the widget heights so the RLV becomes scrollable. In that case the footer will always remain sticky. Correct me if I'm wrong.

Reply:
It will cover that Ananya, RLV dimension is dependent on ScrollView. OnSizeChange, RLV dimension is computed.

Reply:
@manishPatwari there are two components which can change here:
- RLV layout size - which is the size of the container in which the content scrolls. This is the one which is covered by `ScrollComponent`'s `onSizeChanged` and the changes for which have been added.
- Content size - which is the size of the entire scrollable content. Check `LayoutManager`'s `getContentDimension`.

Imo RLV should check and give a callback for scrollable state change any time either of the above two changes. `StickyContainer` should simply listen to this rather than determining on its own whether RLV is scrollable.

Reply:
Done the changes.

Reply:
Awesome, checking